experimental/dds/tree2/src/test/feature-libraries/sequence-field/sequenceFieldToDelta.spec.ts

```diff
@@ -371,3 +376,14 @@ describe("SequenceField - toDelta", () => {
 		});
 	});
 });
+
+// Function to convert an Insert mark to the expected cursors when a changeset is converted to delta.
+function convertInsertMarkToNodeCursors(mark: Insert<never>): ITreeCursorSynchronous[] {
```

Comment:
Added this function to help with getting the cursors for the correct expected delta value, as the new ops encoding return chunkCursors which fails the assert test against the original `contentCursors` which is a `ITreeCursorSynchronous`. 

I see similar tests breaking (e.g. `modularChangeFamilyIntegration.spec.ts`), which I can likely fix in a similar way, but am unsure if this is the approach I should take.

Reply:
removed function and used `assertMarkEqual` and `assertDeltaEqual` instead.